microprofile/WFLY-14940_upgrade-microprofile-health-3.1.adoc

```diff
@@ -0,0 +1,94 @@
+= Upgrade MicroProfile Health to 3.1
+:author:            Martin Stefanko
+:email:             mstefank@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          microprofile,health,observability
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+
+MicroProfile 4.1 provides a minor upgrade for https://github.com/eclipse/microprofile-health/releases/tag/3.1[MicroProfile Health 3.1] with changes described in its https://download.eclipse.org/microprofile/microprofile-health-3.1/microprofile-health-spec-3.1.html#release_notes_3_1[Release Notes]. The MicroProfile Health 3.1 is an incremental, backwards compatible release that only added one major feature called Startup probes (@Startup qualifier and /health/startup endpoint) for the Kubernetes startup probes.
+
+This component upgrade also covers the upgrade of smallrye-health 3.1.1 that implements MicroProfile Health 3.1.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-14940[WFLY-14040]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1763[EAP7-1763]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* mailto:fburzigo@redhat.com[Fabio Burzigotti]
+
+=== Testing By
+
+[x] Engineering
+[ ] QE
+
+=== Affected Projects or Components
+
+* SmallRye Health and MicroProfile Health
```

Comment:
I miss [JBoss EAP OpenShift Templates project](https://github.com/jboss-container-images/jboss-eap-openshift-templates) and more likely WildFly Quickstarts too.
Both of them are mentioned either here or in the TP as going to be affected.

Reply:
I don't see liveness/readiness definitions in templates. Do you know where are these defined?

Reply:
Yep, you need to select a branch in order to see them: https://github.com/jboss-container-images/jboss-eap-openshift-templates/blob/eap74/templates/eap74-basic-s2i.json#L463